vector/src/main/res/layout/fragment_timeline.xml

```diff
@@ -119,8 +129,7 @@
         android:visibility="gone"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        tools:visibility="visible" />
```

Comment:
Please revert this change. Layout editor should show all the views if possible

Reply:
Done. 
Thanks !